build-aux/de.k_bo.Televido.Devel.json

```diff
@@ -44,6 +45,28 @@
     "*.a"
   ],
   "modules": [
+    {
+      "name": "clapper",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dclapper=enabled",
+        "-Dclapper-gtk=enabled",
+        "-Dclapper-app=disabled",
+        "-Dintrospection=enabled",
+        "-Dvapi=enabled"
```

Comment:
Not sure if Rust additionally needs introspection (GIR file), but you surely do not need `vapi` (VALA bindings).

Reply:
Both aren't needed, thanks. I just copied them over from another project.